plugins/module_utils/common.py

```diff
@@ -181,8 +216,16 @@ def auth_set(*names):
         client = get_api_client._pool[digest]
         return client
 
+    def generate_cache_file(kubeclient):
+        import hashlib
+        cache_file_name = 'k8srcp-{0}.json'.format(hashlib.sha1(get_default_cache_id(kubeclient)).hexdigest())
```

Comment:
```suggestion
        cache_file_name = 'k8srcp-{0}.json'.format(hashlib.sha256(get_default_cache_id(kubeclient)).hexdigest())
```

Reply:
sha256 is more secure.

Reply:
Thank you for reviewing @Akasurde. Done.